aspnetcore/get-started.md

```diff
@@ -156,10 +148,10 @@ As soon as you save the file, the running app is updated automatically because y
 
 ## Shut the app down
 
-To shut down the app:
+Follow these steps:
 
 * Close the browser window.
-* Press <kbd>Ctrl</kbd>+<kbd>C</kbd> in the command shell.
+* To shut down the app, press <kbd>Ctrl</kbd>+<kbd>C</kbd> in the command shell.
```

Comment:
Is the change in word order deliberate? "shut down the app" vs. "Shut the app down"

Reply:
Closing the browser window doesn't shut down the app, so I moved it to the bullet point to make it an accurate statement.

I also like having them close the browser window first because the connection to the server will be lost if they shut down the app without closing the window, which could be a little confusing. It's covered in the Blazor node, and we don't want to get into lost connections to the server here.

Reply:
Specifically for word order ... "shut down the app" vs. "shut the app down," I don't think it matters. Did you prefer it the other way?

Reply:
I'll go ahead and merge. If a patch is needed, I can put it on tomorrow's PR for the overview updates.